### PR TITLE
fix: make dep install step as codecell

### PR DESCRIPTION
**Pull request by saihaj**

### CODE REVIEW

File: templates/ray-data-batch-classification/README.ipynb [hunk 1]
```diff
@@ -24,10 +24,22 @@
...
+    "To run this example, you will need the following packages:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pip install -q \"ray[data]\" torch torchvision"
```
Comment (scottjlee) [inline templates/ray-data-batch-classification/README.ipynb:36]: you still need the `!` in code cell to indicate it is bash command
```suggestion
    "!pip install -q "ray[data]" torch torchvision"
```

File: templates/ray-data-batch-classification/README.ipynb [hunk 2]
```diff
@@ -351,8 +363,22 @@
...
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.10.14"
```
Comment (scottjlee) [inline templates/ray-data-batch-classification/README.ipynb:381]: these are metadata changes from running the notebook right? can we drop these if not needed?

File: templates/ray-data-batch-classification/README.ipynb [hunk 3]
```diff
@@ -24,10 +24,22 @@
...
+    "To run this example, you will need the following packages:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install -q \"ray[data]\" torch torchvision"
```
Comment (scottjlee) [inline templates/ray-data-batch-classification/README.ipynb:36]: are the `\` here auto-generated from running the pre-commit? if they are not needed / not auto-generated, can we remove them?

Comment (saihaj) [reply templates/ray-data-batch-classification/README.ipynb:36]: yeah it is auto generating these changes 🤔 I removed it but it still brought it back

**Pull request merged**